### ISSUE TITLE
RHEL 8 with version 0.5.0 and patch #53

### ISSUE THREAD

**Issue by trenta** [hautreux/auks#58]
Hi all,

I'm upgrading our SLURM cluster and wanted to use the latest version of auks too. The first problem I ran into was solved by applying patch #53 

Now on the master node all is working fine but any api commands fail from all other nodes.

```
[trenttest@test-sl01 ~]$ auks -p
Auks API request failed : auks api : request processing failed
[trenttest@test-sl01 ~]$ auks -a
Auks API request failed : auks api : connection failed
```

Here is a verbose auks -p from the same login node https://pastebin.com/wxUV3xYM

With debug and log level both set to 5 on the master node running auksd nothing is logged. The machines are on the same subnet and no firewall involved. They are rebuilds of the same machines I used RHEL 7 and a previous version of auks successfully on.

Any help would be great.

**Comment by trenta** [comment]
I may have found something useful.

New cluster running patched 0.5.0 on RHEL 8 that isn't working
```
lsof -i | grep auks
auksd      1055      root    3u  IPv4  27662      0t0  TCP localhost:italk (LISTEN)
```

Old cluster running patched 0.4.4 on RHEL 7 that is working
```
lsof -i | grep auks
auksd     17147      root    3u  IPv4 115935867      0t0  TCP slurm-cont01.svi.edu.au:italk (LISTEN)
```
I haven't found where this is configured yet.

**Comment by trenta** [comment]
Can't believe it. A new default line in the /etc/hosts file in RHEL 8 that didn't effect anything else. And IPv6 of all things.
